autoextract_poet/mixins.py

```diff
@@ -0,0 +1,19 @@
+from web_poet.mixins import ResponseShortcutsMixin
+from web_poet.page_inputs import ResponseData
+
+
+class AutoExtractResponseShortcutsMixin(ResponseShortcutsMixin):
+
+    @property
+    def response(self):
+        # FIXME: raise a custom Exception or create a separate method for errors
+        data = self.autoextract_response.data
+
+        # FIXME: pageType could be different to item key in the response
```

Comment:
I don't know what I'm saying here :stuck_out_tongue:

Reply:
Ohh, I remember. I'm using pageType here to get the data from the `data` dict. For example, for `product` you have the product within the `product` key. But I'm not sure this is going to be the case for all pageTypes... We have to check the documentation https://doc.scrapinghub.com/autoextract.html

Reply:
The mixin was entirely removed for now.